cstar/roms/simulation.py

```diff
@@ -317,7 +333,12 @@ def __str__(self) -> str:
         if len(self.boundary_forcing) > 0:
             base_str += (
                 f"\nBoundary forcing: <list of {len(self.boundary_forcing)} "
```

Comment:
consider implementing `__str__` or `__repr__` on the type to avoid having to write this manual lookalike `<list of {...}?`